package.json

```diff
@@ -31,6 +31,7 @@
 	"dependencies": {
 		"ip": "^1.1.5",
 		"log4js": ">=0.6.0",
+		"memcached": "^2.2.2",
```

Comment:
Am a bit confused... are we adding memcached as a dependency? Where is its configuration then? And is this already available on production mShot servers?

Reply:
hmm which php code are you talking about when you say:

> PHP checks if there is an entry for the given URL

My instinct is that having memcached as a means of IPC between php and nodejs doesn't seem right, especially if they are already communicating via a REST API.

If memcached is just used for caching, maybe it should be completely external to the nodejs process and be used only at the php level for caching. Or alternately, perhaps the nodejs process should handle both setting and retrieving items from the cache.